nvflare/edge/models/model.py

```diff
@@ -20,7 +22,46 @@
 from torch.nn import functional as F
 
 
-def export_model(net, input_tensor_example, label_tensor_example):
+def export_model_to_bytes(net: nn.Module, input_shape, output_shape):
+    """Exports a PyTorch model to a base64-encoded string suitable for use in embedded or edge environments.
+
+    This function creates dummy input and label tensors based on the provided shapes,
+    runs the model export pipeline (including lowering to Executorch), and returns
+    the serialized model as a base64-encoded string.
+
+    Args:
+        net (nn.Module): The PyTorch model to export.
+        input_shape (tuple): The shape of the input tensor, e.g., (batch_size, channels, height, width).
+        output_shape (tuple): The shape of the output tensor, e.g., (batch_size, num_classes).
+
+    Returns:
+        str: A UTF-8 string containing base64-encoded bytes of the exported model.
+    """
+
+    input_tensor = torch.randn(input_shape)
+    label_tensor = torch.ones(output_shape, dtype=torch.int64)
+    model_buffer = export_model(net, input_tensor, label_tensor).buffer
+    base64_encoded = base64.b64encode(model_buffer).decode("utf-8")
```

Comment:
Should just return model_buffer here to be consistent with the func name.
The caller of this func will convert to b64 str if necessary.

Reply:
good suggestion, changed